include/planes/plane.h

```diff
@@ -209,7 +210,7 @@ void plane_set_pos(struct plane_data* plane, int x, int y);
  * @param scale Scale of the plane, where 1.0 is no scale.
  * @see plane_apply()
  */
-void plane_set_scale(struct plane_data* plane, double scale);
+void plane_set_scale(struct plane_data* plane, double scale_x, double scale_y);
```

Comment:
This is a breaking API change.  Instead, leave the original definition there and add a new one that looks something like this.

`void plane_set_scale_independent(struct plane_data* plane, double scale_x, double scale_y);`

Then make the original plane_set_scale() call your new function with the same value for both parameters.  I'm not incredibly worried about the change in struct plane_data.  I'll add the proper missing getter functions with this change for scale in both dimensions.